src/Mbstring/Mbstring.php

```diff
@@ -523,6 +524,41 @@ public static function mb_strrpos($haystack, $needle, $offset = 0, $encoding = n
         return false !== $pos ? $offset + $pos : false;
     }
 
+    /**
+     * @param string      $string
+     * @param int         $split_length
+     * @param string|null $encoding
+     *
+     * @return array|null|false
```

Comment:
let's remove the dockblock, other functions don't have it and it's not needed as tooling already know the signature of internal function - same for other classes/methods